babel.config.js

```diff
@@ -1,7 +1,10 @@
-module.exports = function (api) {
+export default function (api) {
```

Comment:
Pour pas poser problème de build avec Expo : 
```suggestion
module.exports = function (api) {
```